l10n_it_fatturapa_out_triple_discount/__manifest__.py

```diff
@@ -2,10 +2,10 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'ITA - Fattura elettronica - Integrazione sconto triplo',
-    "summary": "Modulo ponte tra emissione "
-               "fatture elettroniche e sconto triplo",
-    "version": "12.0.2.0.1",
+
+    "name": "ITA - Fattura elettronica - Integrazione sconto triplo",
+    "summary": "Modulo ponte tra emissione " "fatture elettroniche e sconto triplo",
```

Comment:
Queste modifiche non sono necessarie per la migrazione in sé ma solo per **pre-commit**, è possibile spostarle nel commit dedicato?